iOS/Core/FeatureFlag.swift

```diff
@@ -250,7 +250,7 @@ extension FeatureFlag: FeatureFlagDescribing {
         case .networkProtectionRiskyDomainsProtection:
             return  .remoteReleasable(.subfeature(NetworkProtectionSubfeature.riskyDomainsProtection))
         case .experimentalBrowserTheming:
-            return .remoteDevelopment(.feature(.experimentalBrowserTheming))
+            return .remoteReleasable(.subfeature(ExperimentalBrowserThemingSubfeature.onByDefault))
```

Comment:
@amddg44 - can you confirm the RMF setup in https://github.com/duckduckgo/remote-messaging-config/pull/189/files#diff-9dfc4966cd72018379e3965410b0557c9de199105334e60ef5ced8d5b04d6075R1077 will work with the rollout based on the subfeature? Or we need to modify the RMF PR?

Reply:
Confirmed. Details in https://app.asana.com/1/137249556945/project/1209527836175384/task/1210436130003257?focus=true.